File: C#/1005.cs
using System; 

class URI {

    static void Main(string[] args) { 

        double A = double.Parse(Console.ReadLine());
        double B = double.Parse(Console.ReadLine());

        double pesoA = 3.5;
        double pesoB = 7.5;
        
        double media = (A * pesoA + B * pesoB) / (pesoA + pesoB);
        
        Console.WriteLine("MEDIA = " + media.ToString("F5"));
    }

}

File: GO/1005.go
package main

import "fmt"

func main() {
    var notaA, notaB float64
    pesoA, pesoB := 3.5, 7.5

    fmt.Scan(&notaA)
    fmt.Scan(&notaB)

    media := (notaA*pesoA + notaB*pesoB) / (pesoA + pesoB)

    fmt.Printf("MEDIA = %.5f\n", media)
}